autoload/leaderf/python/leaderf/manager.py

```diff
@@ -213,7 +213,21 @@ def _supportsRefine(self):
         return False
 
     def _previewResult(self, preview):
-        pass
```

Comment:
This function is intended to be an interface, I don't want it to have a default behavior.